db/init.sql

```diff
@@ -1,7 +1,5 @@
---tables
-
 CREATE TABLE IF NOT EXISTS artist (
-    artistid SERIAL,
+    artistid INT NOT NULL,
     firstname VARCHAR(25) NOT NULL,
```

Comment:
I'm a bit worried, that the database ("tffgodb") isn't defined in the beginnnig. I think in development it works fine with the docker-compose telling the database name in the environment variables, but we won't have them necessarily in the cloud hosting service. So as an insucrance I would add the connecting to the database with
`\c tffgodb`